README.md

```diff
@@ -15,18 +15,62 @@ It's a package with some (currently two to be exact) tools to use when building
 
 ```
 yarn add @entria/graphql-mongo-helpers
+
+or
+
+npm i @entria/graphql-mongo-helpers
 ```
 
 ### Usage
 
-The most updated source of documentation are the test files, check the files in `src/__tests__/`.
+#### MATCH_1_TO_1
+The example below includes use with or without regex, e sub field.
+```
+const stringToRegexQuery = val => ({ $regex: new RegExp(val) })
+
+const mapping = {
+  code: {
+    type: FILTER_CONDITION_TYPE.MATCH_1_TO_1,
+  },
+  city: {
+    type: FILTER_CONDITION_TYPE.MATCH_1_TO_1,
+    format: stringToRegexQuery
+  },
+  state: {
```

Comment:
we don't handle nested fields yet

can you change?

Reply:
yes, it would be interesting to say that too,